### PR TITLE
PUT endpoints rather than POST

### PR DESCRIPTION
**Pull request by akash1810**
PUT is more sensible for these endpoints because these endpoints are updating an existing atom

### CODE REVIEW

File: conf/routes [hunk 1]
```diff
@@ -7,13 +7,13 @@ GET     /api/atom/:id                   controllers.Api.getMediaAtom(id)
 GET     /api/published-atom/:id         controllers.Api.getPublishedMediaAtom(id)
 GET     /api/config-values              controllers.Api.getConfigValues()
 POST    /api/atom                       controllers.Api.createMediaAtom()
-POST    /api/atom/:id                   controllers.Api.updateMediaAtom(id)
-POST    /api/atom/:id/asset             controllers.Api.addAsset(id)
-POST    /api/atom/:id/revert/:version   controllers.Api.revertAtom(id, version: Long)
+PUT     /api/atom/:id                   controllers.Api.updateMediaAtom(id)
```
Comment (ShaunYearStrong) [inline conf/routes]: These may affect the existing media atom maker.

Comment (akash1810) [inline conf/routes]: yeah... updating it now

Comment (akash1810) [inline conf/routes]: and... done!

**Comment by ShaunYearStrong** [comment]
Sweet 👍

**Pull request merged**